assets/src/domain/eventEditor/services/apollo/mutations/tickets/useOptimisticResponse.ts

```diff
@@ -32,6 +32,7 @@ export const TICKET_DEFAULTS: Ticket = {
 	price: null,
 	prices: null,
 	quantity: -1,
+	registrationCount: 0,
```

Comment:
for brevity we _could_ rename this to `regCount` but that's not a necessity

Reply:
First I did that, then I thought this one is more clear 😄